docs/website/directory.html

```
@@ -490,6 +490,23 @@ <h2 class="section-title" id="devtools">Developer Tools</h2>
 </div>
 </div>
 
+<div class="card" data-cat="devtools" data-tier="a" style="animation-delay:.52s>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)